docs/man_pages/screenbuilder/upgrade-screenbuilder.md

```diff
@@ -0,0 +1,28 @@
+upgrade-screenbuilder
+==========
+
+Usage | Synopsis
+------|-------
+General | `$ appbuilder upgrade-screenbuilder`
+
+Upgrades your project's Screen Builder version.
+<% if(isConsole) { %>WARNING: This command is applicable only to Apache Cordova projects created with Screen Builder.<% } %> 
+
+<% if(isHtml) { %>
+### Command Limitations 
+
+* You can run this command only on projects created with Screen Builder.
```

Comment:
I think that both the console and html helps must state that this is applicable to hybrid projects.